docker/base/Makefile

```diff
@@ -4,9 +4,13 @@ GIT_COMMIT = $(shell git rev-parse --short HEAD)
 GIT_ROOT = $(shell git rev-parse --show-toplevel)
 
 # WARNING: Remember to change the tag when updating the image
-IMAGE_TAG = 1.1.0
+DEPS_HASH = $(shell $(GIT_ROOT)/scripts/gen-deps-hash.sh -d leiningen -d node -d yarn)
```

Comment:
I'm wondering if there could be an easy way to track this via `toolversion` script. Something like:
```bash
scripts/getn-deps-hash.sh --start
scripts/toolversion leiningen
scripts/toolversion node
scripts/toolversion yarn
scripts/getn-deps-hash.sh --end --gethash
```
So we don't have to maintain the list of packages to get hash for manually. But that could be a future improvement.